### ISSUE TITLE
Pseudocompactness-like property

### ISSUE THREAD

**Issue by Moniker1998** [pi-base/data#778]
Lets call (P) to be: every locally finite open cover has a finite subcover.
Its clear that (P) implies pseudocompactness, and for completely regular spaces the converse holds.

Does (P) have a common name in literature, and can we add this property to pi-base?

**Comment by prabau** [comment]
That would be a question for Math stack exchange or mathoverflow.

**Comment by Moniker1998** [comment]
https://mathoverflow.net/questions/479319/pseudocompact-spaces-and-locally-finite-open-covers

alright

theorems I would like to add are as follows

1. (P) implies pseudocompact
2. pseudocompact + completely regular implies (P)
3. paracompact + (P) iff compact

There's also property that's in between that arised: Every locally finite cover is finite. 
We could add this one under some name too if examples arise.

**Comment by StevenClontz** [comment]
N.B. properties that aren't explored in the literature somewhere aren't a great fit for pi-Base, unless there's strong motivation that the addition would improve the database overall (e.g. https://topology.pi-base.org/properties/P000144 helped extend results for non-T_0 spaces)

**Comment by Moniker1998** [comment]
In the literature this property is seen (but not named) in Normal topological spaces by Alo and Shapiro, and also in On pseudo-compact and countably compact spaces by Iseki and Kasahara.

As for motivation, it would add that this query is impossible:
https://topology.pi-base.org/spaces?q=paracompact+%2B+pseudocompact+%2B+regular+%2B+not+compact

**Comment by Moniker1998** [comment]
I just got a response from K.P.Hart, the property $(P_2)$ is called feebly compact.

**Comment by Moniker1998** [comment]
The paper *Herediarily compact spaces* by Stone, or Encyclopedia of general topology list those properties, and more.
But none of the ones other than maybe $P_2$ seem to have a name (i.e. feebly compact).
So perhaps it'd be for the best to add "feebly compact" to pi-base, along with some theorems and examples, and maybe in some distant future to add all the different refinements of pseudocompactness, perhaps as "Stone's property X" if anything.

@prabau @StevenClontz what do you think?

**Comment by prabau** [comment]
Yeah, I think it would be fine to have "feebly compact" in pi-base.  It's a notion that has been studied in the literature and has connections with various other properties.  I'd say no need to add the other ones for now.
